src/types.ts

```diff
@@ -72,7 +72,21 @@ export interface PostFrameUpdateType {
   lastHitBy: number | null;
   stocksRemaining: number | null;
   actionStateCounter: number | null;
+  miscActionState: number | null;
+  isAirborne: boolean | null;
+  lastGroundID: number | null;
```

Comment:
```suggestion
  lastGroundId: number | null;
```
The rest of the SDK uses lowercase `d` in `Id` so we should probably change it to that to be consistent.